controllers/runtimecomponent_controller.go

```diff
@@ -104,22 +103,11 @@ func (r *RuntimeComponentReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	configMap, err := r.GetOpConfigMap(OperatorName, ns)
 	if err != nil {
 		reqLogger.Info("Failed to find runtime-component-operator config map")
-		common.Config = common.DefaultOpConfig()
-		configMap = &corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: OperatorName, Namespace: ns}}
-		configMap.Data = common.Config
+		appstacksutils.CreateConfigMap(OperatorName)
 	} else {
 		common.Config.LoadFromConfigMap(configMap)
```

Comment:
I think we should get rid of this `else` check and move this outside to always load the ConfigMap (i.e. it was just created and we need to load it)

Reply:
I don't think I can do this without modifying appstacksutils.CreateConfigMap(OperatorName)
That function doesn't return the configmap at the moment, so it is possible that we could 'load' an empty map.
It is a bit yuck, but appstacksutils.CreateConfigMap does already copy the default config into common.Config, so I believe the current code is correct.
As previously discussed, I was trying to not change the signature of CreateConfigMap, hence why the code is the way it is. Happy to make further changes if you would prefer.

Reply:
Ahh okay. Missed the default config being set on common.Config. We should be good then. Thanks